src/main/kotlin/com/deviceinsight/helm/DeployMojo.kt

```diff
@@ -167,7 +167,7 @@ class DeployMojo : AbstractMojo() {
 
 			httpClient.execute(request).use { response ->
 				val statusCode = response.statusLine.statusCode
-				if (statusCode != 201) {
+				if ( (statusCode != 200) || (statusCode != 201) ) {
```

Comment:
I'd suggest to accept any response status code in the 2xx range.

```suggestion
				if (statusCode >= 200 && statusCode < 300) {
```